NineChronicles.Headless.Tests/GraphTypes/States/Models/AvatarStateTypeTest.cs

```diff
@@ -66,5 +100,28 @@ public async Task Query(AvatarState avatarState, Dictionary<string, object> expe
                 },
             },
         };
+
+        public static IEnumerable<object[]> CombinationSlotSteteMembers = new List<object[]>()
+        {
+            new object[]
+            {
+                Fixtures.AvatarStateFX,
+                new Dictionary<string, object>
+                {
+                    ["address"] = Fixtures.AvatarAddress.ToString(),
+                    ["combinationSlotState"] = new object[]
+                    {
+                        new Dictionary<string, object>
+                        {
+                            ["address"] = Fixtures.CombinationSlotAddress.ToString(),
+                            ["unlockBlockIndex"] = 0,
+                            ["unlockStage"] = 1,
+                            ["startBlockIndex"] = 0,
+                            ["petId"] = null
+                        }
+                    }
```

Comment:
According to the test case, it seems to filter empty combination slots. Of course, they may always be filled by `CreateAvatar` action's logic.

https://github.com/planetarium/lib9c/blob/86982dddc773a3e2b35a149c9e6749144148b330/Lib9c/Action/CreateAvatar.cs#L159-L164

```csharp
foreach (var address in avatarState.combinationSlotAddresses)
{
    var slotState =
        new CombinationSlotState(address, GameConfig.RequireClearedStageLevel.CombinationEquipmentAction);
    states = states.SetState(address, slotState.Serialize());
}
```

But I think it should include `null` (i.e., there is no state) too because there is no way to present its index (e.g., 1st slot, 2nd slot, 3rd slot etc...) if there is only one slot. To understand, you can see how `query.agent.avatarStates` runs.

And I hope the test case be more detailed for other fields like `unlockBlockIndex` or `startBlockIndex`.